src/test/ui/suggestions/suggest-let-for-assignment.stderr

```diff
@@ -0,0 +1,26 @@
+error[E0425]: cannot find value `x` in this scope
+  --> $DIR/suggest-let-for-assignment.rs:4:5
+   |
+LL |     x = 1;
+   |     ^ not found in this scope
+   |
+help: maybe you meant to write an assignment here
+   |
+LL |     let x = 1;
+   |     ~~~~~
```

Comment:
```suggestion
   |     +++
```
Will it be better if the error is as such?

Reply:
That is controlled by `span_suggestion`...
`let x` is the suggested part, so I think it should be highlighted all.

Reply:
You can use `span.shrink_to_lo()` to get back a span that points at right before `x`, and change the suggestion to be `let `. That way you'll get the `++++` underline.